tests/acceptance/search/search-help-test.ts

```diff
@@ -0,0 +1,100 @@
+import { click, currentURL, visit } from '@ember/test-helpers';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { module, test } from 'qunit';
+import { setupIntl } from 'ember-intl/test-support';
+import { setupOSFApplicationTest } from 'ember-osf-web/tests/helpers';
+import { EnginesIntlTestContext } from 'ember-engines/test-support';
+
+module('Integration | Component | Search help', hooks => {
+    setupOSFApplicationTest(hooks);
+    setupMirage(hooks);
+    setupIntl(hooks, {
+        search_help_test: {
+            heading_1_test: 'Improved OSF Search',
+            heading_2_test: 'OSF Smart Facets',
+            heading_3_test: 'Resources',
+            body_1_test: 'Enter any term in the search box and filter by specific object types.',
+            body_2_test: `Narrow the source, discipline, and more. For example, 
+                find content supported by a specific funder or view only datasets.`,
+            body_3_test: `Remember to add metadata and resources to your own work on OSF to
+                 make it more discoverable!`,
+            skip: 'Skip',
+            next: 'Next',
+            done: 'Done',
+            enumeration_1_test: '1 of 3',
+            enumeration_2_test: '2 of 3',
+            enumeration_3_test: '3 of 3',
+        },
```

Comment:
Are these necessary?

Reply:
Unless we hardcode the values, these are helpful. Additionally, one of your comments was to reduce clutter. This allows the user to see the translation strings used without them being lost in the test. It is also an indirect test of the translations file if however reliant on direct reference.

Reply:
So for every component we have translations we need a copy of the translation strings in the test? I am not sure that's very helpful. You still need to refer to the translation file to see what should be rendered.

I think the test is to make sure that we render a page with elements containing text in the translation file. Whether the text in the translation file is correct is another issue, and perhaps shouldn't be tested here.

To make sure the translation file is correct, one needs only to proofread the translation file itself. Doing this means we need to proofread not only the translation file, but also each one of the test files that do this. I don't think that's something we want to do.

Reply:
'one needs only to proofread the translation file itself.' 

It was only meant to be helpful. If you would like to hardcode the values and only validate for one set of data, I can do that.

Reply:
I'm unresolving this conversation. It looks like we have a test failure stemming from these translation files being mismatched with the actual one we use in the app. I think it is best to remove the translations here.

Reply:
Yes, I just saw it and pushed the change. It just proved having them to be helpful. I can hardcode them if you don't want the additional check but actually like this test.

Reply:
These have now been removed.